sdk/monitor/Azure.Monitor.Query/src/QueryTimeSpan.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Monitor.Query
+{
+    /// <summary>
+    /// Represents a span of time over which the query would be executed.
+    /// </summary>
+    public readonly struct QueryTimeSpan : IEquatable<QueryTimeSpan>
+    {
+        /// <summary>
+        /// Represents the maximum <see cref="QueryTimeSpan"/>.
+        /// </summary>
+        public static QueryTimeSpan MaxValue => new QueryTimeSpan(TimeSpan.MaxValue);
```

Comment:
```suggestion
        public static QueryTimeSpan MaxValue { get; } = new QueryTimeSpan(TimeSpan.MaxValue);
```

I could be mistaken, but the existing form would create a new instance for each call.   Since the type is immutable after construction, should we consider a single instance?